src/api/mocks/rest.ts

```diff
@@ -8,7 +8,7 @@ const createRestHandler = <MethodType extends keyof typeof baseRest>(
   const wrapperFn = ((...params: Parameters<typeof baseRest[MethodType]>) => {
     const [path, resolver] = params;
 
-    const url = new RegExp('^(?:[a-z+]+:)?//', 'i').test(path.toString()) ? path : `/${BASE_URL}${path}`;
```

Comment:
Pipelines suggest that Something is No Yes 🤓

I'm afraid there could have been a reason for the lack of this slash, you can come around to the convent meeting today :D